lms/templates/problem_notifications.html

```diff
@@ -4,9 +4,10 @@
 from openedx.core.djangolib.markup import HTML
 %>
 
-<div class="notification ${notification_type}" tabindex="-1">
+<div class="notification ${notification_type} ${'notification-hint' if hint_notification else 'notification-response'}" tabindex="-1">
```

Comment:
Why can't this be handled by making "hint" one of the notification_type values? 

Perhaps this can be cleaned up in the hint story (I'd rather keep hints out of this story entirely).